From 172e8089ecc79fc67904e7cd8a408a35b2691694 Mon Sep 17 00:00:00 2001
From: Kai Liu <kai.liu@suse.com>
Date: Mon, 6 Jun 2022 19:58:00 +0800
Subject: [PATCH] Revert "Linux 5.10.120"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 70dd2d169d08f059ff25a41278ab7c658b1d2af8.

Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index fdd2ac273f42..b442cc5bbfc3 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 120
+SUBLEVEL = 119
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.1

